packages/cra-external-template/template/.eslintrc.js

```diff
@@ -0,0 +1,82 @@
+module.exports = {
+  env: {
+    browser: true,
+    es6: true,
+    node: true,
+    amd: true,
+    jest: true,
+  },
+  extends: [
```

Comment:
Can we just extend the recommended libs here and bin the rest - if people want to extend they can, our config is quite opinionated